### ISSUE TITLE
Checkbox: Keros tool failure for rule keros-redundant-labels-instructions

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#3721]
#### Describe the issue:

On running keros tool on checkbox input (eg: https://developer.microsoft.com/en-us/fabric#/components/checkbox), the "keros-redundant-labels-instructions" rule fails.

![image](https://user-images.githubusercontent.com/11937034/34936345-0b6eaf0a-fa07-11e7-8280-c8f94ed05629.png)

The button text and explicitly provided label have redundant information.

#### Expected behavior:
There should be no keros tool failures.

**Comment by Jahnp** [comment]
@prmittal, can you please provide more specific details on what the expected fix should be here? I ran the keros tool myself on this page and am not able to reproduce the "keros-redundant-labels-instructions" rule. 

Further, when I inspected the components that are flagged in the background of your screenshot and compare their DOM structure to what's expected for "Native widgets", they don't appear to be non-conforming to the rule. I'm going to go ahead and close this for now as I cannot repro, but please feel free to reopen if you have more details on the issue and if it's affecting your product scenarios.

@betrue-final-final for FYI.
